src/test/java/mate/academy/rickandmorty/ApplicationTests.java

```diff
@@ -6,8 +6,8 @@
 @SpringBootTest
 class ApplicationTests {
 
-	@Test
-	void contextLoads() {
-	}
+    @Test
+    void contextLoads() {
+    }
```

Comment:
```suggestion
        @Test
        void contextLoads() {
        }
```
remove this unnecessary changes